### ISSUE TITLE
The use of the non-specific "endpoint" is confusing

### ISSUE THREAD

**Issue by martinthomson** [DavidSchinazi/draft-cms-masque-connect-ip#12]
Given that the roles are relatively fixed with respect to who can send what here, using "endpoint" makes this hard to follow.  If the idea is to allow a client might assign addresses to a server, then that's an extension.

**Comment by DavidSchinazi** [comment]
This is intentionally bidirectional: in the network-to-network use-case (see [requirements doc](https://datatracker.ietf.org/doc/html/draft-ietf-masque-ip-proxy-reqs-01#section-2.4)), the client might provide the server with an address from the client's pool.
